packages/test/test-utils/src/testContainerRuntimeFactory.ts

```diff
@@ -61,22 +61,25 @@ export const createTestContainerRuntimeFactory = (
 			context: IContainerContext,
 			existing: boolean,
 		): Promise<IRuntime & IContainerRuntime> {
-			const runtime: ContainerRuntime = await containerRuntimeCtor.load(
+			const runtime: ContainerRuntime = await containerRuntimeCtor.loadRuntime({
 				context,
-				[
+				registryEntries: [
 					["default", Promise.resolve(this.dataStoreFactory)],
 					[this.type, Promise.resolve(this.dataStoreFactory)],
 				],
 				// eslint-disable-next-line import/no-deprecated
-				buildRuntimeRequestHandler(
+				requestHandler: buildRuntimeRequestHandler(
 					// eslint-disable-next-line import/no-deprecated
 					defaultRouteRequestHandler("default"),
 					...this.requestHandlers,
 				),
-				this.runtimeOptions,
-				context.scope,
+				provideEntryPoint: () => {
+					throw new Error("TODO: AB#4993");
```

Comment:
Why can't we properly implement this now?  Similar to what `defaultRouteRequestHandler` does?

Reply:
I'd rather have this explicitly handled by the task to remove request pattern in the tests. Throwing an error can quickly let someone know entryPoint isn't yet handled by e2e tests so they don't need to do a ton of debugging.